data_diff/dbt_parser.py

```diff
@@ -81,19 +81,84 @@ def legacy_profiles_dir() -> Path:
     return Path.home() / ".dbt"
 
 
-class TDatadiffModelConfig(pydantic.BaseModel):
+class TDatadiffModelConfig(BaseModel):
```

Comment:
I think we could benefit by moving our pydantic models to a separate module -- like dbt_models.py or something. Non-blocking

Reply:
Updated to a separate module! Good idea as there's enough to justify it.

Reply:
brought it back because I'd have to make too many out of scope changes across the codebase